website/static/js/filepage/index.js

```diff
@@ -198,12 +199,17 @@ var FileViewPage = {
         $('.file-view-panels').removeClass().addClass('file-view-panels').addClass(fileViewPanelsLayout);
 
         m.render(document.getElementById('toggleBar'), m('.btn-toolbar[style=margin-top:20px]', [
-            ctrl.canEdit() ? m('.btn-group', {style: 'margin-left: 0;'}, [
-                m('.btn.btn-sm.btn-danger.file-delete', {onclick: $(document).trigger.bind($(document), 'fileviewpage:delete')}, 'Delete')
+            ctrl.sourceUrl !== 'None' ? m('.btn-group', {}, [
```

Comment:
Wouldn't a ```None``` source_url be set to the string ```'None'``` [here](https://github.com/caseyrollins/osf.io/blob/feature/source-urls/website/templates/project/view_file.mako#L152) first though?